TripPlaner/src/main/java/com/singapore/TripPlaner/Controller/CityController.java

```diff
@@ -0,0 +1,61 @@
+package com.singapore.TripPlaner.Controller;
+
+import com.singapore.TripPlaner.Model.City;
+import com.singapore.TripPlaner.Model.Persistent;
+import com.singapore.TripPlaner.Service.CityService;
+import com.singapore.TripPlaner.Service.dataacces.Reader;
+import com.singapore.TripPlaner.Service.dataacces.Writer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.List;
+
+@Controller
+public class CityController {
+
+    private final CityService cityService;
+    private final Reader reader;
+    private final Writer writer;
+
+    public CityController(CityService cityService, Reader reader, Writer writer) {
+        this.cityService = cityService;
+        this.reader = reader;
+        this.writer = writer;
+    }
```

Comment:
```suggestion
    private final CityService cityService;

    public CityController(CityService cityService) {
        this.cityService = cityService;
    }
```
Reader i Writer powinny być zaimplementowane w cityService, a nie w controlerze